### ISSUE TITLE
The scripts cygdb, cython and cythonize are installed in '/root/.local/bin' which is not on PATH

### ISSUE THREAD

**Issue by billybee** [GoogleCloudPlatform/cloudml-samples#225]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Win 10
- **TensorFlow version (use command below)**: 1.8
- **Python version**: 3.6
- **Exact command to reproduce**:
gcloud ml-engine jobs submit training my_mask_rcnn_201807271234 
    --job-dir=gs://${YOUR_GCS_BUCKET}/train 
    --packages dist/object_detection-0.1.tar.gz,slim/dist/slim-0.1.tar.gz 
    --module-name object_detection.train 
    --region us-central1 
    --runtime-version 1.8 
    --config gs://${YOUR_GCS_BUCKET}/train 
    --train_dir=gs://${YOUR_GCS_BUCKET}/train
    --pipeline_config_path=gs://${YOUR_GCS_BUCKET}/data/pipeline.config

### Describe the problem
I am trying to train my own object detection model using a pre-trained mask rcnn model.  I keep run into issues with ML Engine, and first got errors about packages version too low.  

I then added packages to setup.py 
REQUIRED_PACKAGES = ['Pillow>=1.0', 'Matplotlib>=2.1', 'cython>=0.28.1', 'setuptools==34.0.0', 'requests==2.18.4']

However I still ended up with following error logs and training job was not able to get started

### Source code / logs
You are using pip version 10.0.1, however version 18.0 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

The scripts cygdb, cython and cythonize are installed in '/root/.local/bin' which is not on PATH.
The script chardetect is installed in '/root/.local/bin' which is not on PATH.
Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.

**Comment by happyhuman** [comment]
Hi. Thanks for your message. Before I can assist you any further, can you please tell me which one of the sample code in this repository were you trying to run which led to this issue? I am trying to reproduce the issue on my end as much as possible.

**Comment by nicolamarinello** [comment]
I got the same error. How did you fix it?

**Comment by billybee** [comment]
it is not fixed. but I don't know if it's the root cause of my train job
failure.



2018-08-13 12:03 GMT-06:00 Nicola Marinello <notifications@github.com>:

**Comment by nicolamarinello** [comment]
I'm trying to train an SSD v2 for object detection as described here [https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/running_on_cloud.md](here). I was able on July doing everything in the same way. I would read the old logs to check if this problem is the root of the train job failure but I'm not able to check too old logs

**Comment by akulubala** [comment]
I got this error too, anyone got a solution?

**Comment by wenfuli** [comment]
I got this error too, anyone got a solution?

**Comment by kuldeep27396** [comment]
I got this error too, any solution ?

**Comment by Suadey** [comment]
Had the same error while trying to install gensim library. It was resolved after running the "pip install --user gensim" a couple of times.
